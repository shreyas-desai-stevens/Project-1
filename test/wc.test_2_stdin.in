
a
rh
pxh
zjil
vlkny
sasfig
wabvtgx
epvbwmya
bivaajjfk

t
xx
vwe
xdvp
otzox
wscads
ycclhqj
xbfdibqu
nlmbxaqqg

e
pi
yfk
xcim
bwuil
fehzdg
fyidfom
xhuqespx
pnbrdftjz

f
cc
cdd
dnop
gavpf
cycnnp
qmkforr
ucmdvagg
wcvdjylss

b
bk
vkz
qwfw
tfajx
svkkia
potcbaa
iqgbrnpx
qgvyuupoo

f
le
ind
qxys
djglp
tbzjso
esjjhhc
ftxufgbx
gqkawlrco

x
lh
fva
rltl
niziz
pttqyn
lymztoi
ndyftnqo
bwvlyxwek

u
iy
ssa
echt
omtlh
cyxnqn
tymyqbw
sobjoqyt
drqwonwcd

k
dq
lpf
nohq
odofd
jyucrw
cyzcnqi
knpbntwu
pvfwhaanf

f
yk
blk
zvxa
icfcc
jocidl
dshgkia
dzuizdgk
lcbjtqfzh

r
rc
pxs
ktrm
dgpii
ostqjp
kidntap
wptgzkfb
xbwnghtbg

b
ju
wrz
vlgk
dftnr
trvmtu
obumlkb
mjlwlhup
exhvqitrj

o
oe
tzp
petp
cuxck
kumcio
ptvrwki
bznxxwup
hnszvjscy

a
og
wou
juuo
lpmwc
tngnch
opcfqaq
taetnlkj
exiiqpszt

p
ll
hmn
gbqx
xqozp
kgmjqq
wzjjboi
miudnnmj
dvzcwkgim

r
oa
jgz
zqxl
mowig
xahpfp
mobbpab
hteppruw
ymtcilkzk

t
xi
kjb
tksl
sbmxf
kpycok
eqpqbfg
ejwujbyp
rseezxtui

n
cx
zke
rjsx
zwmml
kxmxor
gvfiolq
izcuhyiy
regaubnya

g
pz
ybk
kvte
hjbqh
vdtmpj
kiswyhb
nefnzwmt
anfhujeql

j
pa
jcf
uvpt
xiudk
geakes
thujyub
ucpzhyjo
psoajsuvg

o
hi
ypc
qvkw
abzgw
hhyfyn
vcnetkf
djpqwaym
oanspvika

c
pg
dsz
wbgz
fypqg
xeuktc
zazhvzj
syapvyrh
xbvnorezz

s
ey
fqn
tccm
wtmew
ddyqas
rszqjbr
ezeefejg
vvsmebmjc

k
ce
kir
gujb
wblmu
iyifeb
miqpeoo
oieggmva
ouyqljzpv

a
ad
ked
hgos
ovyjq
wmzbms
iffdtay
jjtbfmvt
mwbhqevrd

m
cj
qme
ckkn
zshuu
kllsqb
jxeuxqz
eagfxnwj
vjodacyxu

i
ga
mfw
fvnb
jsykc
btpgtm
mjyionn
egjzhtum
ixcyvzovz

o
ud
dgd
hwrp
foiab
odpipn
fgppkat
nphuzjli
oexithyer

o
pb
nwy
idvc
ghdnl
nqcqxe
uzmilgi
aayrofkl
vwexfbvjw

o
od
bym
pfvn
tjwbt
tlxjiu
ssfhsib
kfzpacij
vxchnjcew

o
ne
exh
umgr
zamlu
nizhsv
yqvxnaa
bswksdsc
zrsxvzler

u
ja
tlm
vpzf
kyvph
anovhj
qoetuiy
cdudxijs
qeioglbvk

f
og
qey
gqlv
unhwn
uiynjv
wvupsxw
bgoagcms
yhgduxjjo

a
km
taj
mxgo
zdtto
otzkhm
nioagqx
qvfxkahb
thqancflq

x
sn
jie
ctbx
dvhhi
pqmzgl
mcdpobp
onzhbshw
qaxglzpnk

i
xc
ekn
ctgi
swsvw
lzojgm
pcajjan
ahlzkkny
obrmfelic

d
gi
fti
bgdt
dypct
gtgowd
gprozkd
erndgzfz
ftfmhugtl

r
vt
qts
meoh
fwuub
jmuclz
ydbbmdc
nqbclpmn
mcyodygpt

j
ua
hsz
oceo
qqddi
cpbxoo
qapffjm
schrfxfd
ayrdzjszu

b
ad
cvu
qktj
vteoi
eeidiv
uqgeose
ayjrkzgc
ykygwnqzr

k
rw
twh
koxl
wwmua
xkzaor
fvshmrp
gvivawil
oskdjyqtd

t
gf
xso
rska
itubl
wzgnlw
jckqpjw
mlueayoy
vpmmhmzeo

w
pm
ofp
ahob
wgcgr
acihrd
yaesfaq
qnbqhgef
blolmngtc

i
yi
zne
itkf
qgavk
akicyh
tsmwyil
vewxhedw
nwoqaxswm

p
vo
oru
bzha
ykrel
dpvzcl
uiqwzyu
pumtocsm
vvjwsqkcb

z
hn
ctb
vovs
surlk
jhudzh
izlcuon
mfhxqqgk
zblqitvnc

m
pk
zid
iglr
hovxp
jwsdem
clklppr
vnhlxvco
hwkljyoyf

s
ap
smg
lytf
aklgm
afkicz
dnplaum
vnqzcyte
idefunkam

l
pc
bas
fsgn
wbxge
sxbdoh
jzzqcww
sroznvkc
aatgsnhtv

u
jq
naq
ngnw
wojuo
cggvok
tvrdumq
rjayvofu
xskfnmzfg

h
jk
jhl
hfjh
shoqf
fruqzk
ubseypm
djtlrozn
romohlyuv

i
su
pgr
rwyg
klwte
yzemur
jifudwj
updoxkpu
lsklcgyzz

p
fx
eom
dkqm
kxpkf
nhjpqq
mdgmdsb
txfcyrpm
blwhmrxhg

l
rj
okl
smfl
rrpgo
kylusp
picoyfa
gvbbrbre
aatcnkwfy

n
uh
nij
xhro
ieahm
oqepgy
moesvqh
uawasjzg
ivzyborkg

e
cg
zlw
wjqh
sqnbb
mjtkvm
jwjowth
zptmxngz
omvdgdasd

u
ms
rpb
rqvb
qesxp
owcdvz
mfkhygh
yntabjsm
poxxxmteq

o
gh
yvf
kpqa
zkvde
nuqysa
nkwyvhn
iebnmbie
ytuizraia

k
ka
tcm
mmri
cypaw
ddavcx
spevfih
dcxenrqc
akvnrkqea

d
kl
xnz
kpbi
ggyrq
joaxpr
wkiywis
xwavichp
qaugariwh

b
yj
bks
ngyb
nqumu
qnnzlr
fekrkij
xkewiyuc
knjqiwnun

j
vj
vfe
lxai
jhqbh
wzpwtm
dnipqkd
eoezqkdb
ienkhmpnd

e
xg
dkw
uujr
pixcd
fnfsci
gtetccn
orqeyhcc
pxfgzwrnn

p
fe
anx
sipi
lxavi
djnxtm
qqaijye
fovzyzeq
ctwxljsyx

j
kq
kou
edwg
lwznd
prenqz
aszpiac
vdgfsnft
wzkygzkoq

i
pl
ywc
gbam
mnggy
ygjppn
aklkliz
owiwsxya
caimdqlaz

t
lb
gij
vqbs
kzcth
sgdiea
gvbiesh
rwdkjanr
mhxynifmg

y
rb
ylp
nhfo
iyvxy
cgphgd
mrfcpig
tvbsodug
fiofrajvv

e
ry
arz
rvbw
sodeb
dgpjqb
sbsehpa
ettbrfkj
kelmcpnlp

i
vx
veg
oopw
pojws
niwpdk
whebnse
vnwmnlvx
lakmfpwbk

i
vj
uce
axzv
mwgyi
pgmubl
kzdoece
efmarqrl
ambdzqggs

m
gf
tci
ofjs
erakv
zoysey
rbziwbv
ikllgtkl
mkzuadvaa

c
xa
fsa
geou
kuezv
qvrrgv
vaypcwf
luhzkkew
bpynmczzl

r
ld
lbu
qvjd
eewca
ubztiw
vmoiume
vkdijner
bjljppffr

u
cd
qww
urpk
qulmd
nnptwi
ljblmit
gdjmminh
ntmwqogka

b
ie
mnq
esgo
lhueg
lrwped
yrejynr
zojaswws
iaoiykmek

o
ey
aeq
byvc
isnkn
bzybjq
tuwlfcz
modegxip
mksgnnkab

z
xs
tqc
qlgl
pppty
ledcjl
aqmqqma
ovyadvnp
tjlemhmlg

i
uo
vme
ffpu
wkdgs
pooing
fneiuuc
lafcxdhd
vznnidyds

y
sf
euc
hxyx
xasqn
lznbor
xaplgnb
cftunfah
btveosabt

e
tx
sjb
ietf
mhxer
ppcqey
kvhhuoq
anhvjdpw
avfznmzpf

l
cy
ttn
ipbv
edtih
pfjrno
prjmqzs
dggedpgu
rfdpoqifr

b
ev
nmo
wecz
svypu
trhnhc
lxebhfl
bodaoipl
orcadgkgc

t
mw
qiv
sdbh
cvljo
fcmjee
owfodyf
fmbwvzqh
uycapuild

f
mx
ami
rbgk
uddfd
eybkxa
cmgflgz
isddnxoc
czpmtpaxy

p
ef
kwa
tidn
ruqph
lirjat
ppoqcdh
xzprlstu
jpwdjybag

w
ky
nrv
qxyx
hzbpe
npecep
jqwbgel
umimhjci
gswnukoqi

j
yx
aaq
mpkx
bpnyy
ssjoba
upqdzob
dbuhusqt
movmhzwhe

i
pg
rum
epis
xnlvm
opqfik
nehcytq
eithwhkg
mpywrepku

c
ux
cxm
khcj
orjwf
nkzelj
vmeawxq
cbfzdcpg
oujeptsqw

b
po
pvz
hefg
knxbq
zcxfgu
asjwowl
ckitjypt
oolyogkrw

s
vv
uun
psoh
ygdkq
oceytf
wgawffi
msbrjdyx
wecheoxdn

e
hj
gqr
qcca
yttue
qyiwtx
zbsabol
qubfmjig
gklyeucxq

y
nm
agv
oene
osvut
bcyije
nadfgnu
ngckwmsj
twjtkzgfl

s
ci
ipe
extx
kluva
smldny
bozzuao
cynclxnm
oeuqpiofm

s
hu
nhl
sges
tipld
rbbeiz
uwpgese
nrbehkvc
bnedpyqgp

w
kf
sxa
bkmo
baghm
lkaxxo
jlkofwc
riyzdjwj
uaqdlccsy

j
bd
mcf
wabw
uyqbr
tmbica
xcxsspn
wqjfmbtm
vpxoughnu

h
uw
lxa
bwdg
vcscz
lovcmq
snizvcz
efqlfpyr
wocyzuqvw

v
jn
dio
gnbr
jlhwq
gcmlyy
rozmogk
kequzqry
dfinpxbql
