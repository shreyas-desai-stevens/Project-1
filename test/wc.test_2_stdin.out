1000    900    5500 